README.md

```diff
@@ -131,9 +131,8 @@ environment by monkey-patching.
 [Link to rails docs](http://api.rubyonrails.org/classes/ActiveSupport/Inflector.html)
 
 #### Alternatives
-1. gem: inflecto [link](https://github.com/mbj/inflecto)
-2. gem: linguistics [link](https://github.com/ged/linguistics)
-3. gem: dry-inflector [link](https://github.com/dry-rb/dry-inflector)
+1. gem: linguistics [link](https://github.com/ged/linguistics)
```

Comment:
BTW, this one is last updated 2015 (well, this could mean "it is just feature-complete", but....), and works by extending core classes, just like ActiveSupport, is it really a good alternative to suggest?..

Reply:
Not sure that it's a good idea to drop this like in current PR. But I'll be happy to delete it 👍